.changes/next-release/enhancement-Configure-43901.json

```
@@ -0,0 +1,5 @@
+{
+  "type": "enhancement",
+  "category": "Configure",
```

This is a nit, but for consistency with existing changelog messages this should be a lowercase `configure`, like these:

- https://github.com/aws/aws-cli/blob/a95b915dd21b39862930d5c4d21c9caf63e45079/CHANGELOG.rst#L1997
- https://github.com/aws/aws-cli/blob/a95b915dd21b39862930d5c4d21c9caf63e45079/CHANGELOG.rst#L8983

```suggestion
  "category": "configure",
```